spec/policies/wiki_policy_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe WikiPolicy do
+
+  let(:user) { User.new }
```

Comment:
`user` is being assigned correctly in each test that needs it, meaning this `let` line (line 5) isn't being used. Line 5 can be removed.